test/sponsors-test.js

```diff
@@ -22,9 +22,53 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-import {loadSponsorsFromString} from '../lib/sponsors';
+import {loadSponsorsFromString, parse} from '../lib/sponsors';
```

Comment:
Weird that the pre-commit checks do not fix this before pushing

Reply:
which part?

Reply:
oh I see, the expanded view in GH has warnings about "at least one empty line"